CHANGELOG.md

```diff
@@ -4,6 +4,11 @@
 
 * Improve robustness of file type detection by considering the actual MIME type of the content. [lp:1445885](https://bugs.launchpad.net/mixxx/+bug/1445885) [#4356](https://github.com/mixxxdj/mixxx/pull/4356) [#4357](https://github.com/mixxxdj/mixxx/pull/4357)
```

Comment:
I have tested this fix and updated the corresponding issue on launchpad. We have to move the entry to the 2.4.0 section, because the fix in 2.3.2 only works partially depending on the capabilities of the registered SoundSources.

Reply:
Fix for main: https://github.com/mixxxdj/mixxx/pull/4608/commits/00d4f9afdf6d9af02f096d536cd4384dd0d104b8

Just delete this line. Should not cause any merge conflicts then.